x/nft-transfer/types/packet.go

```diff
@@ -0,0 +1,82 @@
+package types
+
+import (
+	"strings"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var (
+	// DefaultRelativePacketTimeoutHeight is the default packet timeout height (in blocks) relative
+	// to the current block height of the counterparty chain provided by the client state. The
+	// timeout is disabled when set to 0.
+	DefaultRelativePacketTimeoutHeight = "0-1000"
```

Comment:
could you explain what's the first value `0` means? Is that always 0 or it can be the other number?

Reply:
`<revision_number>-<revision_height>`